main/main.cpp

```diff
@@ -1225,7 +1226,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			}
 
 		} else if (I->get() == "--export-release" || I->get() == "--export-debug" ||
-				I->get() == "--export-pack") { // Export project
+				I->get() == "--export-pack" || // Export project
+				I->get() == "--import") { // Import
+			audio_driver = NULL_AUDIO_DRIVER;
+			display_driver = NULL_DISPLAY_DRIVER;
+
```

Comment:
Is there anyone who's using `--export` without `--headless`? If not, I'd like to reduce the number of CLI argument permutations that need to be supported.

Reply:
We tried making `--export` imply `--headless` (or `--no-window` in `3.x`) a while ago, but it caused various regressions. Some of them are still present such as https://github.com/godotengine/godot/issues/55711 and https://github.com/godotengine/godot/issues/66842.